client/src/components/SubmitButton.js

```diff
@@ -0,0 +1,19 @@
+import styled from '@emotion/styled';
+
+const SubmitButton = styled.button`
+  width: 136px;
+  height: 46px;
+  border-radius: 20px;
+  background-image: linear-gradient(#504481, #1e194f);
+  color: #d05888;
+  font-size: 17px;
+  font-family: 'Roboto', sans-serif;
+  font-weight: 300;
+
```

Comment:
You could also think about adding `outline: none` to get rid of the blue border when clicking on the buttton :)

Reply:
oh right. thank you for the advice! I forgot 😅